### PR TITLE
Adding doc for sdc metrics

### PR DESCRIPTION
**Pull request by vballu**

### CODE REVIEW

File: commons/tools/metrics.md [hunk 1]
```diff
@@ -89,17 +91,18 @@ vb -> verb
 path -> path
 bIn -> bytesInt
 bOut -> bytesOut
-h -> hostname
+h -> hostname (missing for addons)
```
Comment (KannarFr) [inline commons/tools/metrics.md]: s/missing for addons/available for applications/

File: commons/tools/metrics.md [hunk 2]
```diff
@@ -89,17 +91,18 @@ vb -> verb
...
 rTime -> responseTime
 sTime -> serviceTime
 scheme -> scheme
 sC -> statusCode
 sT -> statusText
 tS -> Haproxy termination_state
-adc -> Reverse proxy hostname
+adc | sdc -> Reverse proxy hostname
```
Comment (KannarFr) [inline commons/tools/metrics.md]: Maybe reprecise in parenthesis that adc is for apps, and sdc for addons.

File: commons/tools/metrics.md [hunk 3]
```diff
@@ -89,17 +91,18 @@ vb -> verb
...
 w -> workerId (Sozu)
 r -> requestId (Sozu)
 tlsV -> tlsVersion (Sozu)
+sDuration -> for SDC only. Total session duration time in millis (haproxy tcp)
```
Comment (KannarFr) [inline commons/tools/metrics.md]: Maybe something like "-> total session duration time in millis (for addons. Look at rTime for applications)"

File: commons/tools/metrics.md [hunk 4]
```diff
@@ -60,15 +60,17 @@ All your applications access logs are pushed to [Warp10](/doc/tools/warp10/). Yo
 
 Access logs are defined in the `'accessLogs'` Warp10 class and there are three Warp10 labels available:
 
-* owner_id;
-* app_id;
-* adc (reverse proxy used).
+* `owner_id`: Organisation ID
+* `app_id` or `addon_id`: Application ID or Addon ID
+* `adc` or `sdc` : `adc` (Application Delivery Controller) are used for http connexions ; `sdc` (Service Delivery Controller) are used for tcp connexions.
```
Comment (KannarFr) [inline commons/tools/metrics.md]: Maybe just add new list (more readable) like:

* adc or sdc:
   * adc:
   * sdc:

**Comment by urcadox** [comment]
Maybe we should have two lists? There are quite a few more missing properties for logs emitted by a sdc (namely: vb, path, h, rTime, sTime, sC and sT)

**Comment by cnivolle** [comment]
@vballu There are changes requested by @KannarFr.

Then we'll merge :)

File: commons/tools/metrics.md [hunk 5]
```diff
@@ -60,15 +60,23 @@ All your applications access logs are pushed to [Warp10](/doc/tools/warp10/). Yo
 
 Access logs are defined in the `'accessLogs'` Warp10 class and there are three Warp10 labels available:
 
-* owner_id;
-* app_id;
-* adc (reverse proxy used).
+* `owner_id`: Organisation ID
+* `app_id` or `addon_id`: Application ID or Addon ID
+* `adc` or `sdc`
+    * `adc` (Application Delivery Controller) are used for http connexions
```
Comment (KannarFr) [inline commons/tools/metrics.md]: s/connexions/connections/, and maybe s/http/HTTP/.

File: commons/tools/metrics.md [hunk 6]
```diff
@@ -60,15 +60,23 @@ All your applications access logs are pushed to [Warp10](/doc/tools/warp10/). Yo
...
+    * `sdc` (Service Delivery Controller) are used for tcp connexions
```
Comment (KannarFr) [inline commons/tools/metrics.md]: s/connexions/connections/, and maybe s/tcp/TCP/.

File: commons/tools/metrics.md [hunk 7]
```diff
@@ -102,6 +110,40 @@ r -> requestId (Sozu)
...
+  ct -> city
+  co -> country
+bIn -> bytesInt
+bOut -> bytesOut
+scheme -> scheme
+tS -> Haproxy termination_state
+sdc -> Reverse proxy hostname
+w -> workerId (Sozu)
+r -> requestId (Sozu)
+tlsV -> tlsVersion (Sozu)
```
Comment (KannarFr) [inline commons/tools/metrics.md]: Not available for addons.

File: commons/tools/metrics.md [hunk 8]
```diff
@@ -102,6 +110,40 @@ r -> requestId (Sozu)
...
+ipD -> ipDestination
+pD -> portDestination # 0 if undefined
+d -> destination
+  lt -> latitude
+  lg -> longitude
+  ct -> city
+  co -> country
+bIn -> bytesInt
+bOut -> bytesOut
+scheme -> scheme
```
Comment (KannarFr) [inline commons/tools/metrics.md]: Not available for addons.

File: commons/tools/metrics.md [hunk 9]
```diff
@@ -102,6 +110,40 @@ r -> requestId (Sozu)
...
+  co -> country
+ipD -> ipDestination
+pD -> portDestination # 0 if undefined
+d -> destination
+  lt -> latitude
+  lg -> longitude
+  ct -> city
+  co -> country
+bIn -> bytesInt
+bOut -> bytesOut
```
Comment (KannarFr) [inline commons/tools/metrics.md]: Not available for addons.

**Pull request merged**